### PR TITLE
Add tentative syllabus.

### PR DESCRIPTION
**Pull request by yungyuc**
This pull request is created for discussing the syllabus.  @jjhuang314 and @tjwei, could you comment?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+machine image of Ubuntu Linux that is loaded with the necessary software
+packages and the workshop materials.
+
+## Instructors:
+
+- Albert Huang
+- Tzer-jen Wei
+- [Yung-Yu Chen](mailto:yyc@solvcon.net)
+
+## (Tentative) Contents
```
Comment (yungyuc) [inline README.md:27]: This section lists the topics that "I" thought we can cover.  @jjhuang314 and @tjwei, your inputs are mandatory before we can move on.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+- Ufunc
+  - Performance
+- Linear algebra
+- Discrete Fourier transform
+
+### NumPy C Extension
+
+- Cython
+
+### Problems (Hands-on)
```
Comment (yungyuc) [inline README.md:58]: I think hands-on problem solving should be blended into the above topics.  But at this early stage I don't know how to do it yet.

File: README.md (hunk 1 above)
Comment (tjwei) [inline README.md:27]: will do it this week.

**Comment by tjwei** [comment]
seems fine to me.

**Comment by tjwei** [comment]
I merge the tentative syllable into the master, so that we can further develop it and discuss sub issues.

**Pull request merged**